packages/google_sign_in/google_sign_in_web/example/pubspec.yaml

```diff
@@ -12,12 +12,16 @@ dependencies:
     path: ../
 
 dev_dependencies:
+  build_runner: ^2.1.1
   flutter_driver:
     sdk: flutter
   flutter_test:
     sdk: flutter
+  google_identity_services_web: ^0.2.0
   google_sign_in_platform_interface: ^2.2.0
   http: ^0.13.0
   integration_test:
     sdk: flutter
+  jose: ^0.3.3
```

Comment:
I "need" this to generate a fake JDK token that can be decoded by the plugin later.

I could potentially just hardcode the signed token for the test for now so we can remove this dependency, but then generating new test cases is going to be more cumbersome.

Reply:
I removed this by hardcoding the values of the JWT tokens, and linking to the site that can be used to generate more. I deleted the code that I used to generate them, since more can be generated by decoding the one that is in the file, and modifying it on the website.